src/test/java/uk/gov/ons/fwmt/job_service/rest/impl/JobResourceServiceImplTest.java

```diff
@@ -2,23 +2,16 @@
 
 import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-import uk.gov.ons.fwmt.job_service.ApplicationConfig;
+import org.mockito.InjectMocks;
 import uk.gov.ons.fwmt.job_service.rest.JobResourceService;
 import uk.gov.ons.fwmt.job_service.rest.dto.JobDto;
 
 import static org.junit.Assert.*;
 
-@RunWith(SpringRunner.class)
-@SpringBootTest(classes = {ApplicationConfig.class})
 @Ignore
 public class JobResourceServiceImplTest {
```

Comment:
does this test run?

Reply:
That class im testing next and will be changes to correct layout